### PR TITLE
homework v.7

### PR DESCRIPTION
**Pull request by dkh**

### CODE REVIEW

File: homework.rb [hunk 1]
```diff
@@ -1,6 +1,14 @@
-VARIANT = 0
+VARIANT = 7
 
 def generate
   # put your solution here (delete the line below)
-  rand(1000000).to_s + '-' + rand(1000000).to_s
+  letters = []
+ 
+  2.times do |i|
+  	i = ('A'..'Z').to_a.sample(3).join
```
Comment (sveredyuk) [inline homework.rb]: 2 spaces

File: homework.rb [hunk 2]
```diff
@@ -1,6 +1,14 @@
...
+  	letters << i
+  end
+ 
+  letters[0] + '-' + rand(1000000..9999999).to_s + '-' + letters[1]
```
Comment (sveredyuk) [inline homework.rb]: you interpolate result + call + result... not ruby way.

move create rand number to var and interpolate it.

Comment (dkh) [reply homework.rb]: If i insert rand number to the array, will it be right?

File: homework.rb [hunk 3]
```diff
@@ -2,13 +2,12 @@
...
+  2.times do |letters|
+    letters = ('A'..'Z').to_a.sample(3).join
+    result << letters
   end
- 
-  letters[0] + '-' + rand(1000000..9999999).to_s + '-' + letters[1]
-
+  numbers = rand(1000000..9999999).to_s
+  result.insert(1, numbers)
+  result.join('-')
```
Comment (sveredyuk) [inline homework.rb]: and now try without result variable...

**Comment by sveredyuk** [comment]
@geekdkh tests not passed

File: homework.rb [hunk 4]
```diff
@@ -1,6 +1,12 @@
...
   # put your solution here (delete the line below)
-  rand(1000000).to_s + '-' + rand(1000000).to_s
+  letters = []
+  2.times do |letter|
+    letter = ('A'..'Z').to_a.sample(3).join
+    letters << letter
+  end
+  numbers = rand(1000000..9999999).to_s
+  letters[0] + '-' + numbers + '-' + letters[1]
 end
```
Comment (sveredyuk) [inline homework.rb]:
```ruby
# good ruby code style... hope so :) check test before commit
VARIANT = 7
LETTERS_RANGE = 'A'..'Z'
NUMBERS_RAMGE = 1000000..9999999

def letters
  LETTERS_RANGE.to_a.sample(3).join
end

def numbers
  rand(NUMBERS_RAMGE).to_s
end

def generate
  "#{letters}-#{numbers}-#{letters}"
end
```

**Review by sveredyuk — APPROVED** [review]

**Pull request closed without merging**